core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -11,11 +11,10 @@
 
 package alluxio;
 
+import javax.annotation.concurrent.ThreadSafe;
```

Comment:
It is doesn't metter.BTW, you can use idea plugin checkstyle and findbug to check your code use  alluxio/build/checkstyle/alluxio_checks.xml, they will help you more.